go.mod

```diff
@@ -165,7 +167,7 @@ require (
 	github.com/DataDog/opentelemetry-mapping-go/pkg/quantile v0.20.0
 	github.com/DataDog/sketches-go v1.4.6
 	github.com/DataDog/viper v1.13.5
-	github.com/DataDog/watermarkpodautoscaler v0.6.1
+	github.com/DataDog/watermarkpodautoscaler v0.5.3-0.20241011111846-034635582ee1
```

Comment:
This is the commit of DataDog/watermarkpodautoscaler#221.

To be updated as soon as this PR is merged.